src/lib/components/collections/collections-list/searchTree/Request.svelte

```diff
@@ -2,29 +2,26 @@
   import { replaceSlashWithGreaterThanSymbol } from "$lib/utils/helpers/common.helper";
   import { getMethodStyle } from "$lib/utils/helpers/conversion.helper";
   import { handleRequestClick } from "$lib/utils/helpers/handle-clicks.helper";
-  export let path: string = "";
+  export let explorer: any;
```

Comment:
We can make a interface to specify it's type, also we can change the name to something else which resonates to it's use.